src/Components/ImagesTable/ImagesTable.js

```diff
@@ -175,7 +205,20 @@ const ImagesTable = () => {
             Create image
           </Link>
           <EmptyStateSecondaryActions>
-            <DocumentationButton />
+            <Button
+              component="a"
+              target="_blank"
+              variant="link"
+              icon={<ExternalLinkAltIcon />}
+              iconPosition="right"
+              isInline
+              href={
+                'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/creating_customized_rhel_images_using_the_image_builder_service/index'
```

Comment:
This seems to redirect here, I wonder if we should use the redirected URL?
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/creating_customized_images_by_using_insights_image_builder/index

Reply:
Good point, the original url behaves a bit strange. Updated!